### PR TITLE
BaSP-TG-59: Projects LIST/DELETE - Redux implementation

### PR DESCRIPTION
**Pull request by NancyCima**
In this branch the following tasks were done:

- [x] Install Redux Dev Tools

- [x] Set Actions and Constants

- [x] Set Reducers

- [x]  Implement Thunks

- [x] Implement List

- [x]  Implement Delete

NOTE: The total work on this entity is shared with Ricardo Dosko, who is in charge of doing the implementation for Add and Edit.

### CODE REVIEW

File: src/Components/Projects/List/List.jsx [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
+import Row from '../../Shared/Row/Row';
+import { useSelector, useDispatch } from 'react-redux'; //useSelector
+import { deleteProject } from '../../../redux/projects/thunks';
+
+const List = ({
+  //list,
+  //setShowModal,
+  //setTitleModal,
+  //deleteItem,
+  //setLoading,
```
Comment (javierbiselli) [inline src/Components/Projects/List/List.jsx:13]: borrar estos comments

File: src/Components/Projects/index.js [hunk 2]
```diff
@@ -1,84 +1,89 @@
...
-    try {
-      const response = await fetch(`${process.env.REACT_APP_API_URL}/projects`);
-      const data = await response.json();
-      setList(data.data);
-    } catch (error) {
-      console.error(error);
-    }
-  };
+  const [showForm, setShowForm] = useState(false);
+  //const [loading, setLoading] = useState(true);
```
Comment (javierbiselli) [inline src/Components/Projects/index.js:20]: borrar los useState que ya no usas

File: src/Components/Projects/index.js [hunk 3]
```diff
@@ -1,84 +1,89 @@
...
-    </section>
+    <>
+      {isLoading ? (
+        <Loader show={true} />
+      ) : (
+        <section className={styles.container}>
+          <List
+            //setShowModal={setShowModal}
+            //setTitleModal={setTitleModal}
+            //deleteItem={deleteItem}
```
Comment (javierbiselli) [inline src/Components/Projects/index.js:56]: lo mismo aca

**Review by LucianoAlarcon — APPROVED** [review]

**Review by EzeRocco — APPROVED** [review]

**Review by CristianAlcober — APPROVED** [review]
Hi Nancy! I think everything works as intended, I don't see any problems at the moment. I'm waiting to see everything merged, with all the options available. Keep it up!

**Pull request merged**